pkgs/development/tools/clpm/default.nix

```diff
@@ -4,7 +4,9 @@
 , wrapLisp
 # Broken on newer versions:
 # https://gitlab.common-lisp.net/clpm/clpm/-/issues/51
-, sbcl_2_0_8
+# Once that bug is fixed, replace this with regular ‘sbcl’ and remove all
+# references to sbcl 2.1.9 from nixpkgs, including from sbcl/2.x.nix.
+, _sbcl_2_1_9
```

Comment:
If this is the only package left to rely on another sbcl version, it could be justified to write something like this:
```nix
let
  # Broken on newer versions:
  # https://gitlab.common-lisp.net/clpm/clpm/-/issues/51
  sbcl = sbcl.overrideAttrs (_: {
    src = fetchurl {
      url = "mirror://sourceforge/project/sbcl/sbcl/2.1.9/sbcl-2.1.9-source.tar.bz2";
      sha256 = "189gjqzdz10xh3ybiy4ch1r98bsmkcb4hpnrmggd4y2g5kqnyx4y";
    };
  });
in ...
```
This could allow us to remove the rest of this version mapping already without having to wait for that bug to be fixed and released.

Reply:
How about this? Because the `wrapLisp` function itself doesn't provide an .override, you can directly access the `.override` provided by callpackage on the underlying sbcl call to 2.x.nix. Long story short: you can just import 2.1.9 directly in clpm, from 2.x.nix, without going through all-packages. This gets us half-way there.

Reply:
Sure, this looks like an improvement to me.